remmanager.go

```diff
@@ -3,15 +3,17 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 )
 
 // confirmDelete listens for user confirmation and returns a boolean
-// ! untested
-func confirmDelete(projName string) bool {
+// Takes in an input channel to increase testability
+// * tested
+func confirmDelete(projName string, stdin io.Reader) bool {
 	fmt.Printf("Are you sure you want to delete %s? ('y' or 'n')\n>", projName)
-	reader := bufio.NewReader(os.Stdin)
+	reader := bufio.NewReader(stdin)
```

Comment:
Why is the new way preferable over the original way? Idk much about os.Stdin vs passing in an io.Reader

Reply:
The functionality for Canaveral CLI is exactly the same. No advantage there.

However, when testing confirmDelete(), I can create and pass in my own buffer stream for canaveral to read from and write to. This is basically piping it away from stdin and into my own bufstream thing. I needed to do this to track the output of the function as well as simulate user input.

Reply:
ahhh gotcha ty